src/containers/SessionContainer.test.tsx

```diff
@@ -23,6 +23,7 @@ jest.mock('../client/apolloClient', () => ({
     },
   },
 }))
+jest.mock('uuid/v4', () => () => '1')
```

Comment:
is this needed now when we did `__mocks__.*`?

Reply:
Doubt it! Lets check
```suggestion
```